src/experimental/shader-modules/fp64/math-fp64.glsl.js

```diff
@@ -17,6 +17,7 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+export default `\
```

Comment:
> can you copy the latest version of math-fp64.glsl.js from deck.gl here?

@shaojingli Sorry missed this comment when I merged. Will get it updated in a separate PR.